src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/SqlIaaSExtensionTests.ps1

```diff
@@ -227,10 +227,10 @@ function Test-SetAzureRmVMSqlServerExtension
     finally
     {
         # Cleanup
-        if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
-        {
+        #if(Get-AzureRmResourceGroup -Name $rgname -Location $loc)
```

Comment:
Commented code should be removed. Since the `finally` has no code, it should be removed too.